helm/superset/values.yaml

```diff
@@ -160,7 +160,7 @@ extraConfigMountPath: "/app/configs"
 
 image:
   repository: apache/superset
-  tag: latest
+  # tag: "v{{ $.Chart.AppVersion }}"
```

Comment:
Can we just use this templated value? I think that adding the ability to template these fields would be useful.
i.e.:
```
tag: "v{{ $.Chart.AppVersion }}"
```
and then in deployment template, etc:
```
image: "{{ .Values.image.repository }}:{{ tpl .Values.image.tag . }}"
```

Reply:
BTW, I'm not sure that the referencing  `"v{{ $.Chart.AppVersion }}"` will work at all, since Helm values don't support any templating besides `include` and `template` via plane YAML AFAIR

Reply:
Could be, but it's worth a try :). I've found that in general, the more fields that support templating, the better.